magma/docker/app/docker-entrypoint.sh

```diff
@@ -3,23 +3,31 @@ set -e
 
 export PATH="/app/node_modules/.bin:/app/vendor/bundle/$RUBY_VERSION/bin:$PATH"
 
+function findEnvConfig() {
```

Comment:
We'll have to standardize that `test` config comes second, right?

Reply:
Yeaaaa... you're not wrong.  I could use `yq` by adding it into the container, I just avoided that because it's an additional download into the container, but maybe that's a better approach in the end.

Reply:
> So I don't think this is an issue explicitly with this PR, but I'm now getting an error on the Apache container (I can file a separate issue if needed). This happens after running `docker image prune -a` to clean out old images, and then `make up`.
> 
> ```
> Building apache
> Step 1/4 : FROM library/httpd:2.4.43
> 2.4.43: Pulling from library/httpd
> Digest: sha256:387f896f9b6867c7fa543f7d1a686b0ebe777ed13f6f11efc8b94bec743a1e51
> Status: Downloaded newer image for httpd:2.4.43
>  ---> ccbcea8a6757
> Step 2/4 : WORKDIR /app
>  ---> Using cache
>  ---> d48c83ddef7a
> Step 3/4 : RUN apt-get update
>  ---> Using cache
>  ---> d38067d54fc4
> Step 4/4 : RUN apt-get install -y libapache2-mod-xsendfile
>  ---> Running in 49966eb4db03
> Reading package lists...
> Building dependency tree...
> Reading state information...
> E: Unable to locate package libapache2-mod-xsendfile
> ERROR: Service 'apache' failed to build: The command '/bin/sh -c apt-get install -y libapache2-mod-xsendfile' returned a non-zero code: 100
> make[3]: *** [../make-base/docker-compose.mk:17: build] Error 1
> ```
> 
> Do you think there's a version conflict somewhere or is the image misconfigured, since it says it's a newer version of `httpd`? What version are you running?

That's, super odd.  I'm using an explicit version, so httpd's version did not change.  Unfortunately, I am installing xsendfile via apt with these two commands

```
RUN apt-get update
RUN apt-get install -y libapache2-mod-xsendfile
```

Which I suppose is non deterministic if the apt index is not working.  Try running in the root monoetna dir `docker-compose build --no-cache` just to see if there's some issue with that.

Reply:
Oh I see, yeah it says from your output that a newer image was downloaded.  That would mean they publish a new image for the existing version, which is odd...